### PR TITLE
HC-933 Clean up Bluetooth articles

### PR DESCRIPTION
**Pull request by seaxwi**
Correct trademark usage and improve content.

**Changes proposed in this pull request**
- Correct trademark usage
- Improve/prune articles

**Please check**
- [x] This pull request changes an article title
- [x] This pull request removes an article

### CODE REVIEW

**Comment by seaxwi** [comment]
"Boards and shields with wireless connectivity"

can replaced by

"Which Arduino boards support BLE?"

**Review by Renat0Ribeir0 — APPROVED** [review]
Looks good. I'm not sure if the trademark symbol is needed every time the word Bluetooth is mentioned. If not, it is ready to merge.

File: content/Hardware/Generic/Boards-and-shields-with-wireless-connectivity.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+* MKR VIDOR 4000
+* MKR WiFi 1010
+* Nano 33 IoT
+* UNO WiFi Rev 2
+* Portenta H7
+* Portenta H7 Lite Connected
+
+## Boards with Bluetooth<sup>®</sup> connectivity
+
+These boards support Bluetooth Low Energy **and/or** what's now known as Bluetooth _Classic_.
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Hardware/Generic/Boards-and-shields-with-wireless-connectivity.md:18]: No, we use it with the "first and most prominent" mention. The symbol should be in superscript (enclosed by <sup> tags) except in the article title, which does not support this.

File: content/Hardware/Generic/Change-the-broadcast-name-of-a-Bluetooth-peripheral.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Change the broadcast name of a Bluetooth® peripheral"
+---
+
+When using the [ArduinoBLE](https://www.arduino.cc/reference/en/libraries/arduinoble/) library to enable Bluetooth Low Energy communication on an Arduino board, the broadcasted name is set by the [setLocalName()](https://www.arduino.cc/reference/en/libraries/arduinoble/setlocalname/) function.
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Hardware/Generic/Change-the-broadcast-name-of-a-Bluetooth-peripheral.md:5]: https://github.com/arduino/help-center-content/pull/8#discussion_r824645440

**Pull request merged**